docs/config/README.md

```diff
@@ -317,16 +317,16 @@ The module is only visible when the device's battery is below 10%.
 
 ### Options
 
-| Option               | Default                           | Description                                         |
-| -------------------- | --------------------------------- | --------------------------------------------------- |
-| `full_symbol`        | `""`                             | The symbol shown when the battery is full.          |
-| `charging_symbol`    | `""`                             | The symbol shown when the battery is charging.      |
-| `discharging_symbol` | `""`                             | The symbol shown when the battery is discharging.   |
-| `unknown_symbol`     | `""`                             | The symbol shown when the battery state is unknown. |
-| `empty_symbol`       | `""`                             | The symbol shown when the battery state is empty.   |
-| `format`             | `"[$symbol$percentage]($style) "` | The format for the module.                          |
-| `display`            | [link](#battery-display)          | Display threshold and style for the module.         |
-| `disabled`           | `false`                           | Disables the `battery` module.                      |
+| Option               | Default                            | Description                                         |
+| -------------------- | ---------------------------------- | --------------------------------------------------- |
+| `full_symbol`        | `""`                              | The symbol shown when the battery is full.          |
+| `charging_symbol`    | `""`                              | The symbol shown when the battery is charging.      |
+| `discharging_symbol` | `""`                              | The symbol shown when the battery is discharging.   |
+| `unknown_symbol`     | `""`                              | The symbol shown when the battery state is unknown. |
+| `empty_symbol`       | `""`                              | The symbol shown when the battery state is empty.   |
+| `format`             | `"[$symbol $percentage]($style) "` | The format for the module.                          |
```

Comment:
I think all the other modules handle this kind of thing by adding the space to `$symbol` instead of the format string. Could you do it like this here too?

Reply:
Sure.